clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriter.java

```diff
@@ -490,6 +500,24 @@ public void ingestSSTFiles(RocksDB rocksDB, List<ColumnFamilyHandle> columnFamil
     }
   }
 
+  public void createSnapshot(RocksDB rocksDB) {
+    try {
+      Checkpoint checkpoint = makeCheckpoint(rocksDB);
+      String checkpointPath = this.fullPathForPartitionDBSnapshot;
+
+      if (Files.exists(Paths.get(checkpointPath))) {
```

Comment:
What's the circumstances the snapshot directory already exist? Is it the second push? If so, what's the expected behavior?

Reply:
I misread some codes when I made the comment and forgot to delete this.
I meant to ask when we have new pushes, would the content under the same path get overwritten? If yes, there's no concern

Reply:
removed this